packages/background/src/permission-interactive/service.ts

```diff
@@ -162,6 +163,28 @@ export class PermissionInteractiveService {
     );
   }
 
+  async ensureEnabledForBitcoin(env: Env, origin: string): Promise<void> {
+    await this.keyRingService.ensureUnlockInteractive(env);
+
+    // TODO: support ledger
+    // await this.ensureKeyRingLedgerAppConnected(env, "Bitcoin");
+
+    const currentChainIdForBitcoin =
+      this.permissionService.getCurrentChainIdForBitcoin(origin) ?? [
+        `bip122:${GenesisHash.MAINNET}:taproot`,
+        `bip122:${GenesisHash.MAINNET}:native-segwit`,
+      ];
```

Comment:
```suggestion
    const currentChainIdForBitcoin =
      this.permissionService.getCurrentChainIdForBitcoin(origin) ?? `bip122:${GenesisHash.MAINNET}:taproot`;
```
여기는 string이 와야할 것 같아요